documentation/book/proc-bridge-retrieving-latest-messages-from-consumer.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge-quickstart.adoc
+
+[id='proc-bridge-retrieving-latest-messages-from-consumer-{context}']
+= Retrieving the latest messages from a Kafka Bridge consumer
+
+Next, retrieve the latest messages from a Kafka Bridge consumer by requesting data from the link:https://strimzi.io/docs/bridge/latest/#_poll[records^] endpoint. For production use, HTTP clients can retrieve messages repeatedly or as a one-off operation. 
+
+.Prerequisites
+
+* The Kafka Bridge is deployed to your Kubernetes cluster.
+* The Kafka Bridge Service is exposed to your local machine on port 8080.
+* The address of the Kafka Bridge to use as the `<bridge-address>` parameter.
+* The Kafka Bridge consumer is subscribed to a topic. 
+
+.Procedure
+
+* Retrieve messages from the Kafka Bridge consumer (these messages were sent to the consumer in xref:proc-producing-messages-from-bridge-topics-partitions-{context}[Producing messages to topics and partitions]):
++
+[source,curl,subs=attributes+]
+----
+curl -X GET <bridge-address>/consumers/bridge-quickstart-consumer-group/instances/bridge-quickstart-consumer/records \
+  -H 'accept: application/vnd.kafka.json.v2+json'
+----
++
+If the request was successful, the Kafka Bridge returns an array of messages -- describing the topic name, key, value, partition, and offset -- in the response body, along with a `200` code. Messages are retrieved from the latest offset by default.
++
+[source,json,subs=attributes+]
+----
+INSERT JSON HERE
```

Comment:
Is this still some TODO? Or is this actually expected in the document? Seems like a bug.

Reply:
Apologies, this is still to do. I need to verify all requests and responses in my local cluster.